src/editor/codemirror/lint/lint.ts

```diff
@@ -71,6 +70,10 @@ export interface Diagnostic {
   /// An optional array of actions that can be taken on this
   /// diagnostic.
   actions?: readonly Action[]
+  /// Tags that influence the display of the 
+  /// Currently supported tags include `"unnecessary"` and `"deprecated"`
+  /// which are formatted with opacity and strikethrough respectively.
```

Comment:
Doesn't need the first line of explanation?

Reply:
Looks like I got distracted halfway through the sentence. Fixed in 7cdef5aea7ce6730696802886f24bcb2abb4c229.